privacyidea/static_new/src/app/components/token/token-card/token-tab/token-tab.component.ts

```diff
@@ -220,4 +212,33 @@ export class TokenTabComponent {
       )
       .subscribe();
   }
+
+  unassignSelectedTokens() {
+    const selectedTokens = this.tokenSelection();
+    this.dialog
+      .open(ConfirmationDialogComponent, {
+        data: {
+          serial_list: selectedTokens.map((token) => token.serial),
```

Comment:
whats the naming convention for angular? CamelCase or snake_case? There is also numberOfTokens in data